src/main/pg/dyn_notch_config.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "platform.h"
+
+#ifdef USE_DYN_NOTCH_FILTER
+
+#include "pg/pg_ids.h"
+
+#include "dyn_notch_config.h"
+
+PG_REGISTER_WITH_RESET_FN(dynNotchConfig_t, dynNotchConfig, PG_DYN_NOTCH_CONFIG, 0);
+
+void pgResetFn_dynNotchConfig(dynNotchConfig_t *config)
```

Comment:
Just as a remark, for simple cases like this one it is also possible to use a reset template instead of a reset function, see https://github.com/betaflight/betaflight/blob/9e6fd18630f48914484db908867be12265284077/src/main/pg/stats.c#L30-L37 for an example.